### DISCUSSION TITLE
can I enable MQTT?

### DISCUSSION THREAD

**Discussion by jaeti01** [Blackymas/NSPanel_HA_Blueprint#1387] [General]
Hi,
I am pretty new to this project. Thanks for running it!
So far I have one panel running with nspanel-lovelace-UI and now I have setup  a second one with the blueprint approach to figure out what suits me more. 
First question I have: Can I enable the ESPHome MQTT code and have certain values directly written to my broker?
I am a bit hesitant as the sample yaml code says "##### DO NOT CHANGE ANYTHING! #####"
thank you!

**Comment by edwardtfn** [comment]
> First question I have: Can I enable the ESPHome MQTT code and have certain values directly written to my broker?

We are not using mqtt with this project, just the ESPHome native APIs, which could be used directly or you can find almost everything on Home Assistant.

> I am a bit hesitant as the sample yaml code says "##### DO NOT CHANGE ANYTHING! #####" 

It's not that bad as it could sound... It's a group of non-native English speakers doing almost everything on this project, so sometimes we are not successful of transmitting our thoughts.

Anyway, that area when the "don't touch" message shouldn't be changed in most of the cases, but certainly will have exceptions.
We have even an entire page about customizations: https://github.com/Blackymas/NSPanel_HA_Blueprint/wiki/(EN)-Customization and are pretty friendly with people asking for non-standard things. 😉

**Reply by edwardtfn** [reply]
It's made to be easy for the ones without coding skills and flexible to advanced users.

**Reply by jaeti01** [reply]
Thanks for getting back to me that fast. I am also not a native English speaker, so don't worry ;-).
I'll dig into the customisation page. Thanks for the reference.

**Reply by jaeti01** [reply]
Oh - the MQTT requirement btw stems from the fact that I have a Wild West of Home automation tools which in the long run want to migrate to HA. But for the time being MQTT is what glues them together.
